lib/src/connector/connection.cc

```diff
@@ -343,7 +343,7 @@ class verbose_verification {
         X509_NAME_oneline(X509_get_subject_name(cert), subject_name, 256);
         X509_NAME_oneline(X509_get_issuer_name(cert), issuer_name, 256);
         bool verified = verifier_(preverified, ctx);
-        LOG_TRACE("Verifying %1%, issued by %2%. Verified: %3%",
+        LOG_DEBUG("Verifying %1%, issued by %2%. Verified: %3%",
```

Comment:
@Iristyle sorry, I missed the notifications for the above messages :(
Will look into this.